build-aux/modules/50-libdatachannel.json

```diff
@@ -12,10 +12,9 @@
     "sources": [
         {
             "type": "git",
-            "url": "https://github.com/paullouisageneau/libdatachannel.git",
+            "url": "https://github.com/murillo128/libdatachannel.git",
```

Comment:
I see that this fork is 24 versions behind. Is there some specific patch that's not merged back upstream that is needed?

Reply:
I can switch back to upstream now! Completely forgot about that  :/

Are you building something with OBS+WebRTC @mogren I had no idea!?

Reply:
Sorry, not really, just reading code trying to learn. 🐰 🕳️ 🕳️ everywhere... 😁